### PR TITLE
[phi] tranfer the selu_op and pass the CI

### PR DESCRIPTION
**Pull request by 2742195759**
### PR types

Others
### PR changes

Others
### Describe

tranfer the selu_op and pass the CI

### CODE REVIEW

**Review by YuanRisheng — COMMENTED** [review]
[paddle/phi/kernels/impl/.selu_grad_kernel_impl.h.swp]，这个文件是误提交上去的吧？

**Review by MingMingShangTian — DISMISSED** [review]
LGTM

File: paddle/phi/kernels/gpu/selu_grad_kernel.cu [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "paddle/phi/backends/gpu/gpu_context.h"
+#include "paddle/phi/core/kernel_registry.h"
+#include "paddle/phi/kernels/impl/selu_grad_kernel_impl.h"
+#include "paddle/phi/kernels/selu_grad_kernel.h"
```
Comment (zyfncg) [inline paddle/phi/kernels/gpu/selu_grad_kernel.cu]: `"paddle/phi/kernels/selu_grad_kernel.h"` 放在开头，这里可能是pre-commit自动调整了

File: paddle/phi/kernels/gpu/selu_kernel.cu [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "paddle/phi/backends/gpu/gpu_context.h"
+#include "paddle/phi/core/kernel_registry.h"
+#include "paddle/phi/kernels/impl/selu_kernel_impl.h"
+#include "paddle/phi/kernels/selu_kernel.h"
```
Comment (zyfncg) [inline paddle/phi/kernels/gpu/selu_kernel.cu]: 同上

**Review by zyfncg — DISMISSED** [review]

**Review by YuanRisheng — APPROVED** [review]

**Pull request merged**